yt_dlp/extractor/bilibili.py

```diff
@@ -676,6 +680,9 @@ def _real_extract(self, url):
         aid = video_data.get('aid')
         old_video_id = format_field(aid, None, f'%s_part{part_id or 1}')
         cid = traverse_obj(video_data, ('pages', part_id - 1, 'cid')) if part_id else video_data.get('cid')
+        if is_festival or not self.is_logged_in:
+            query = {'try_look': 1} if not self.is_logged_in else {}
```

Comment:
this seems redundant if we are also popping `try_look` from the query params in `_download_playinfo()`